src/reflect/scala/reflect/internal/util/Statistics.scala

```diff
@@ -267,19 +308,20 @@ quant)
    */
   final val hotEnabled = false
```

Comment:
I've decided to reuse `canEnable` for `hotEnabled` as well, so that when we do measure statistics we get the whole information. Let me know if you're ok with this.